### ISSUE TITLE
Can't accept collection offers - error "You must supply the appropriate criterias for criteria based items"

### ISSUE THREAD

**Issue by Rotemy** [ProjectOpenSea/opensea-js#692]
1.I had to fetch the collection orders via the API
2. I had to call the fulfillOrder like this
`await openseaSDK.fulfillOrder({
            order: {
                protocolData: order.protocol_data,
                protocolAddress: order.protocol_address
            },
            recipientAddress: accounts[0],
            accountAddress: accounts[0]
        });`
3. I get "You must supply the appropriate criterias for criteria based items"

Basically the API doesn't return consideration criteria.

**Comment by XingyuLi20** [comment]
+1

**Comment by omnigolp** [comment]
@Rotemy 
Have you deserialized the order with the function from the sdk?

**Comment by Rotemy** [comment]
> @Rotemy Have you deserialized the order with the function from the sdk?

I opened this issue before the new API for collection offers. I need to test it with the new API.

**Comment by mustafa222lk** [comment]
works now ?
